backend/test/controllers/user.test.js

```diff
@@ -0,0 +1,53 @@
+const { getAll, getUser, addUser, updateUser, deleteUser } = require("../../src/controllers/user");
```

Comment:
I'm hoping your thoughs here was to add test to the existing, and as we change them, that we add mocks, and update the tests.  although i'm not sure i would have went so heavy on the tests.